packages/orbit-design-tokens/package.json

```diff
@@ -26,10 +26,13 @@
     "build:xml": "NODE_RUN=true babel-node ./scripts/generateXMLDesignTokens.js",
     "prebuild": "rimraf lib src/js",
     "build": "yarn clean && yarn run build:lib && yarn run generate-json && yarn run build-html && yarn run build-web && yarn run build-ios && yarn run build-android && yarn build:xml",
+    "quickbuild": "yarn build:lib",
```

Comment:
why you need that `quickbuild` script? can't you just use the `yarn build:lib`?

Reply:
I named it so that the intention is a bit more obvious. Doesn't help much, but I thought it's at least something.

Reply:
i think it's not necessary 🤔